File: 0125-valid-palindrome/0125-valid-palindrome.py
class Solution(object):
    def isPalindrome(self, str):
        """
        :type s: str
        :rtype: bool
        """
        s = "".join( c.lower() for c in str if c.isalnum() )

        i = 0
        j = len(s) -1
        while(i <= j) :
            if( s[i] != s[j] ) :
                return False
            i+=1
            j-=1
        return True

File: 0125-valid-palindrome/0125-valid-palindrome.cpp
class Solution {
public:
    bool isPalindrome(string str) {
        string s = "";

        for(char c : str) {
            if(isalnum(c)) {
                s += tolower(c);
            }
        }
        int i = 0;
        int j = s.length() - 1;

        while(i<=j) {
            if(s[i] != s[j]) {
                return false;
            }
            i++ ;
            j-- ;
        }
        return true;
    }
};